week1/index.js

```diff
@@ -1,25 +1,62 @@
-// To run this file: node index.js
-// Then in your browser: http://localhost:8080
+// Determine the port and the initial value of state
+var state = 10;
+var port = 8080;
```

Comment:
Does the port change during runtime? If not, this might be a good use of `const`.